src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star:nth-child(-n),
```

Comment:
The nth-child selector '-n' in '.stars--0 .stars__star:nth-child(-n)' is incorrect. This selector does not select any elements. If you intend to select no stars, you should remove this rule or use a different approach to achieve the desired effect.